File: src/types/enums/Format.ts
export enum EnumFormat {
    SIX = "SIX",
    FOUR = "FOUR",
    TWO = "TWO"
};

export const FormatLabels: Record<EnumFormat, string> = {
    [EnumFormat.SIX]: '6x6',
    [EnumFormat.FOUR]: '4x4',
    [EnumFormat.TWO]: '2x2',
};

File: src/main/java/com/blockout/notifications/models/enums/Format.java
package com.blockout.notifications.models.enums;

public enum Format {
    SIX("6x6"),
    FOUR("4x4"),
    TWO("2x2");

    private final String label;

    Format(String label) {
        this.label = label;
    }

    public String getLabel() {
        return label;
    }
}